tf_helpers/tf_jax_stax.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Stax is a small but flexible neural net specification library from scratch.
```

Comment:
Please update description to your specific project

Reply:
Solved in https://github.com/google/neural-tangents/pull/63/commits/b45f975929b3121a9c6dcb20274fcb097b4e4337